src/bindings/app/git.rs

```diff
@@ -44,6 +44,29 @@ fn git_commit(repo: &str, message: &str, sig: Option<(String, String)>) -> Resul
     Ok(())
 }
 
+fn git_clone(url: &str, into: &str) -> Result<(), git2::Error> {
+    debug!("Cloning into: {}", into);
+	let _repo = match git2::Repository::clone(url, into) {
+	    Ok(repo) => repo,
+	    Err(e) => panic!("failed to clone: {}", e),
```

Comment:
The problem with the code is that you are using a panic. I would not advise using a panic in such a manner, but instead since there is a `Result` just return the error instead and let the binding handle the error, or just use the `?` operator making it be `git2::Repository::clone(url, into)?` (assuming that `git2::Repository::clone` returns a `git2::Error`). Fix these and it would be ready to be merged